### PR TITLE
set qps and burst for agent rest config

### PR DESCRIPTION
**Pull request by ldpliu**
https://issues.redhat.com/browse/ACM-5096

### CODE REVIEW

**Comment by ldpliu** [comment]
Will set the log level in another pr

File: agent/cmd/agent/main.go [hunk 1]
```diff
@@ -156,7 +163,10 @@ func parseFlags() *config.AgentConfig {
 		"The configuration file for the kubernetes event exporter")
 	pflag.BoolVar(&agentConfig.EnableGlobalResource, "enable-global-resource", false,
 		"Enable the global resource feature.")
-
+	pflag.Float32Var(&agentConfig.QPS, "qps", 150,
+		"qps for leaf hub")
```
[the remark this answers is not in the document]
Comment (ldpliu) [reply agent/cmd/agent/main.go]: Done

File: operator/main.go [hunk 2]
```diff
@@ -283,7 +283,7 @@ func getElectionConfig(kubeClient *kubernetes.Clientset) (*commonobjects.LeaderE
 	}
 
 	configMap, err := kubeClient.CoreV1().ConfigMaps(hubofhubsconfig.GetDefaultNamespace()).Get(
-		context.TODO(), operatorconstants.ControllerLeaderElectionConfig, metav1.GetOptions{})
+		context.TODO(), operatorconstants.ControllerConfig, metav1.GetOptions{})
```
[the remark this answers is not in the document]
Comment (ldpliu) [reply operator/main.go]: Changed the code, and only get the configmap once.

**Review by clyang82 — APPROVED** [review]

**Pull request merged**